app/src/main/java/org/loklak/wok/utility/Constants.java

```diff
@@ -7,6 +7,8 @@
 
 public class Constants {
 
+    public static final String BASE_URL_LOKLAK = "https://api.loklak.org/";
```

Comment:
@SKrPl , I think there is a need of config file where some parameters like `BASE_URL_LOKLAK` can be set so that this can be changed whenever needed. This can be done in subsequesnt PRs.

Reply:
@vibhcool In android projects config files are not used for parameters that are used in the Project (the business logic), usually, they are defined in a Constants class. Config files are used to define parameters which are required for building the project, or to keep some API related configurations which should not be shared like for Fabric and Google APIs.